kratos/utilities/process_factory_utility.h

```diff
@@ -1,10 +1,12 @@
-// KRATOS  ___|  |                   |                   |
-//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
-//             | |   |    |   | (    |   |   | |   (   | |
-//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
```

Comment:
What is the purpose of this utility?
I thought we should preferably not pass the python objects to c++?
Overall it seems quite hackish

Reply:
When you remesh you lost most of the information, including the BCs. This utility allows to run again the processes after remeshing and restore that information lost. This is used in the contact mechanics adaptative strategy right now

Reply:
Not really my call but I think this could be task of the stage to redo the processes (Since it owns the processes)

Reply:
Yes, for a normal remeshing you can do that (the one currently in the MeshingApplication), but in this case we use a convergence criteria, then the remeshing is done in C++, so we need to be able to access the processes from C++

Reply:
i agree with the comment of @philbucher, furthermore the BCs are applied (includ ing fixity) in the InitializeSolutionStep and unfixed in the FinalizeSolutionStep EXACTLY with the purpose of allowing remeshing.

most importantly you do not own the processes, so in case you want to do this you would have to make a "remeshing stage" and call functions from there